docs/source/contributing/developing.adoc

```diff
@@ -150,6 +150,12 @@ If you see any Dep errors during `make vendor`, you can remove local cached dire
 [[build-minishift-binary]]
 === Building the minishift Binary
 
+Before running build, you need to run
+
+----
+$ go get -u github.com/go-bindata/go-bindata/...
```

Comment:
No no no... the script already does this, but places it in your `$GOPATH` and `$GOPATH/bin`. You just need to make sure you set `export PATH=$PATH:$GOPATH/bin` to use this...